src/components/AddButton/AddButton.scss

```diff
@@ -0,0 +1,48 @@
+@import "src/styles/variables/_variables";
+@import "src/styles/Icon";
+
+$borderColor: $linkColor;
+
+.addButton {
+  @extend .icon;
```

Comment:
I don't recommend use of `@extend ` better use placeholder in sass `%` or `mixin`.

More info here :
https://www.sitepoint.com/avoid-sass-extend/
https://tiffanybbrown.com/2017/03/dont-use-extend-sass/index.html